src/ios/CDVMobilePayAppSwitch.m

```diff
@@ -134,12 +133,12 @@ - (void)handleMobilePayPaymentWithUrl:(NSURL*)url
         inflightPaymentCallbackId = nil;
         inflightOrderId = nil;
 
-    } error:^(NSError * _Nonnull error) {
+    } error:^(MobilePayErrorPayment * _Nullable error) {
```

Comment:
If the errors are now nullable, doesn't that mean it will crash below on the property access?

Reply:
I'm not sure why this was changed to nullable, and whether or not they have to be (I'm not an iOS developer), but [here is where I got it from](https://github.com/welrachid/dk-wsy-ntamobilepay-public/commit/a69613938827c52e08135dd43f0ee45a868a97d7#diff-ae9b87ca01d1e16832e393e43e089e1f). I don't have a mac here, so could you maybe check if it's okay to change to Nonnull?